### PR TITLE
Fix scripts on windows

### PR DESCRIPTION
**Pull request by MKaHead**
Fixes scripts on windows

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "[Check out our Contribution File](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#rules-and-lintings)",
   "module": "lib",
   "scripts": {
-    "build": "NODE_ENV=production npm run build-components",
+    "build": "cross-env NODE_ENV=production npm run build-components",
```
Comment (markus-walther) [inline package.json:7]: Good catch!

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006 -s src/static",
     "build-storybook": "build-storybook -s ./src/static",
-    "test": "npm run test-lint && prettier --check '.' && node ./scripts/build.test.js && npm run test-ui",
+    "test": "npm run test-lint && prettier --check . && node ./scripts/build.test.js && npm run test-ui",
```
Comment (markus-walther) [inline package.json:17]: I hope prettier checks recursively, like eslint?!

File: scripts/build.test.js [hunk 3]
```diff
@@ -84,7 +84,8 @@ const allFiles = glob.sync(`src${sep}components${sep}*${sep}**${sep}*.js`);
 
 allFiles.forEach(file => {
   // file is like this: src/components/20-molecules/datepicker/index.js
-  const { 2: category } = file.split(sep);
+  // file looks even on windows like that, thats why necessary hardcode the slash
+  const { 2: category } = file.split('/');
```
Comment (markus-walther) [inline scripts/build.test.js]: This is super-hard to read!
Better: `const [_src, _components, category] = file.split('/');`

File: scripts/build.test.js [hunk 4]
```diff
@@ -84,7 +84,8 @@ const allFiles = glob.sync(`src${sep}components${sep}*${sep}**${sep}*.js`);
 
 allFiles.forEach(file => {
   // file is like this: src/components/20-molecules/datepicker/index.js
-  const { 2: category } = file.split(sep);
+  // file looks even on windows like that, thats why necessary hardcode the slash
```
Comment (markus-walther) [inline scripts/build.test.js]: Wording: ... that's why **it is** necessary **to** hard-code

**Review by markus-walther — CHANGES_REQUESTED** [review]
Hi Marco,

good work, just small issues to improve!

You may merge on your own without re-approval after you fixed them.

Cheers, Markus

File: package.json (hunk 2 above)
Comment (MKaHead) [reply package.json:17]: yes

File: scripts/build.test.js [hunk 5]
```diff
@@ -84,7 +84,9 @@ const allFiles = glob.sync(`src${sep}components${sep}*${sep}**${sep}*.js`);
 
 allFiles.forEach(file => {
   // file is like this: src/components/20-molecules/datepicker/index.js
-  const { 2: category } = file.split(sep);
+  // file looks even on windows like that, that's why it is necessary to hard-code the slash
```
Comment (markus-walther) [inline scripts/build.test.js:87]: Good now!

**Review by markus-walther — APPROVED** [review]
All good now, Marco!

**Pull request merged**